config/config_mgmt.py

```diff
@@ -0,0 +1,709 @@
+#!/usr/bin/env python
+#
+# config_mgmt.py
+# Provides a class for configuration validation and for Dynamic Port Breakout.
+
+try:
+
+    # import from sonic-cfggen, re use this
```

Comment:
This library read\writes in config DB, using the functions in sonic-cfggen and in ConfigDBConnector. 

Here, I am importing sonic-cfggen using load_source. I think, that is the only way to import it, since it is stored without .py extension, let me know if you think a better way exists.  Thx.